pages/application.tsx

```diff
@@ -83,185 +63,190 @@ const Application: NextPage = () => {
+							</Box>
+						</Stepper.Step>
+						<Stepper.Step label="Prior Experience">
+							<Box sx={{ maxWidth: 600 }} mx="auto">
+								<Stack>
+									<NativeSelect
+										required
+										label="T-Shirt Size"
+										{...form.getInputProps('shirtSize')}
+										data={['XS', 'S', 'M', 'L', 'XL', 'XXL']}
+									/>
+									<MultiSelect
+										label="Dietary Restrictions"
+										{...form.getInputProps('dietRestrictions')}
+										data={dietOptions}
+										searchable
+										creatable
+										getCreateLabel={(query) => query}
+										onCreate={(query) => {
+											const item = { value: query, label: query };
+											setDietOptions([...dietOptions, { value: query, label: query }]);
+											return item;
+										}}
+									/>
+									<NativeSelect
+										label="How many hackathons have you participated in?"
+										{...form.getInputProps('hackathonCount')}
+										data={[
+											{ value: 'N', label: 'My first hackathon!' },
+											{ value: 'S', label: '1-3' },
+											{ value: 'M', label: '4-6' },
+											{ value: 'L', label: '7+' }
+										]}
+									/>
+									{/* TODO: FileInput has a Capture prop has type boolean | "user" | "environment"
+										Add after we know hosting server and resume site api */}
+									<FileInput
+										label="Upload your resume. PDF only!"
+										description="Resume is optional, but highly encouraged for your application."
+										{...form.getInputProps('resume')}
+										placeholder="Submit resume"
+										accept="application/pdf"
+									/>
+									<TextInput label="LinkedIn" {...form.getInputProps('linkedin')} />
+									<TextInput label="GitHub" {...form.getInputProps('github')} />
+									<MultiSelect
+										label="Other Sites"
+										{...form.getInputProps('otherSites')}
+										data={otherURLs}
+										searchable
+										creatable
+										getCreateLabel={(query) => query}
+										onCreate={(query) => {
+											setOtherURLs([...form.values.otherSites, query]);
+											return query;
+										}}
+										rightSection={null}
+									/>
+									<Switch label="I am looking for a team" {...form.getInputProps('lookingForTeam')} />
+									<Switch label="I am interested in attending the PreHacks event" {...form.getInputProps('attendingPrehacks')} />
+									<Group position="right" mt="md">
+										<Button
+											variant="default"
+											onClick={() => {
+												setStep(step - 1);
+											}}>
+											Back
+										</Button>
+										<Button
+											onClick={() => {
+												if (!form.validate().hasErrors) {
+													setStep(step + 1);
+												}
+											}}>
+											Next
+										</Button>
+									</Group>
+								</Stack>
+							</Box>
+						</Stepper.Step>
+						<Stepper.Step label="Consent">
+							<Box sx={{ maxWidth: 600 }} mx="auto">
+								<Stack>
+									<Checkbox
```

Comment:
Although these checkboxes are functionally required, there is no asterisk by their label explicitly indicating them as such before you press submit. They aren't strictly necessary so it's fine without it, but if you can do it than we can put that fix on this pull request.

Reply:
there wasnt really a good way to put asterisks built in, so i added spans to the labels, let me know if this is good or i should revert it

Reply:
The spans look good. 👍